web/packages/teleport/src/Locks/types.ts

```diff
@@ -54,38 +59,43 @@ export type AllowedTargets =
   | 'access_request'
   | 'device';
 
+/**
+ * TargetValue is the value of the target resource that a lock is applied to.
+ * For example, if a TargetResource is 'node', its corresponding TargetValue should be
+ * the node's UUID. If a TargetResource is 'role', its corresponding TargetValue should be
+ * its name.
```

Comment:
If there's a central canonical reference elsewhere in the codebase that decides this, it could be linked here.

Reply:
There is not, hence the bug :-\